src/test/java/teammates/test/cases/logic/CoursesLogicTest.java

```diff
@@ -374,7 +375,8 @@ public void testGetCourseDetails() throws Exception {
         ______TS("null parameter");
 
         try {
-            coursesLogic.getCourseDetails(null);
+            String nullableCourseId = null;
```

Comment:
Suggest renaming NullableCourseId to nullCourseId. Nullable implies that null is a valid state for the course id.